gensim/test/test_sklearn_api.py

```diff
@@ -1151,6 +1159,58 @@ def testModelNotFitted(self):
         self.assertRaises(NotFittedError, phrases_transformer.transform, phrases_sentences[0])
 
 
+class TestPhrasesTransformerCommonTerms(unittest.TestCase):
+    def setUp(self):
+        numpy.random.seed(0)
```

Comment:
That's not good, because this freeze "global" seed (effect to all test cases, not only to your class). Why you need this here? Probably you can simply remove this line.

Reply:
Quite honestly, I was just looking at the other test cases. Almost all of them include that line in `setUp`. But I'll remove it. :-)

Reply:
oh, you are right, that's looks like a bug for me 
```
ivan@P50:~/release/gensim$ find gensim/ -name "*.py" | xargs grep "numpy.random.seed"
gensim/test/test_sklearn_api.py:        numpy.random.seed(0)  # set fixed seed to get similar values everytime
gensim/test/test_sklearn_api.py:            id2word=dictionary, num_topics=2, passes=100, minimum_probability=0, random_state=numpy.random.seed(0)
gensim/test/test_sklearn_api.py:            id2word=dictionary, num_topics=10, passes=100, minimum_probability=0, random_state=numpy.random.seed(0)
gensim/test/test_sklearn_api.py:            minimum_probability=0, random_state=numpy.random.seed(0)
gensim/test/test_sklearn_api.py:        numpy.random.seed(0)  # set fixed seed to get similar values everytime
gensim/test/test_sklearn_api.py:        model = LdaTransformer(num_topics=2, passes=10, minimum_probability=0, random_state=numpy.random.seed(0))
gensim/test/test_sklearn_api.py:            minimum_probability=0, random_state=numpy.random.seed(0)
gensim/test/test_sklearn_api.py:        numpy.random.seed(0)  # set fixed seed to get similar values everytime
gensim/test/test_sklearn_api.py:        numpy.random.seed(0)  # set fixed seed to get similar values everytime
gensim/test/test_sklearn_api.py:        numpy.random.seed(13)
gensim/test/test_sklearn_api.py:        numpy.random.seed(0)  # set fixed seed to get similar values everytime
gensim/test/test_sklearn_api.py:        numpy.random.seed(0)
gensim/test/test_sklearn_api.py:        numpy.random.seed(0)  # set fixed seed to get similar values everytime
gensim/test/test_sklearn_api.py:        numpy.random.seed(0)
gensim/test/test_sklearn_api.py:        numpy.random.seed(0)  # set fixed seed to get similar values everytime
gensim/test/test_sklearn_api.py:        numpy.random.seed(0)
gensim/test/test_sklearn_api.py:        lda_model = LdaTransformer(num_topics=2, passes=10, minimum_probability=0, random_state=numpy.random.seed(0))
gensim/test/test_sklearn_api.py:        numpy.random.seed(0)
gensim/test/test_sklearn_api.py:        lda_model = LdaTransformer(num_topics=2, passes=10, minimum_probability=0, random_state=numpy.random.seed(0))
gensim/test/test_sklearn_api.py:        numpy.random.seed(0)
gensim/test/test_sklearn_api.py:        numpy.random.seed(0)
gensim/test/test_sklearn_api.py:        numpy.random.seed(0)

```

that's not your fault, please simply remove this line from your code, that's enough.